src/ucp/rndv/proto_rndv.inl

```diff
@@ -318,4 +318,17 @@ ucp_proto_rndv_recv_complete(ucp_request_t *req)
     return ucp_proto_rndv_recv_complete_status(req, rreq->status);
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_proto_rndv_receive(ucp_worker_h worker, ucp_request_t *rreq,
+                       const ucp_rndv_rts_hdr_t *rndv_rts_hdr,
+                       const void *rkey_buf, size_t rkey_length)
+{
+    if (worker->context->config.ext.proto_enable) {
+        ucp_proto_rndv_receive_start(worker, rreq, rndv_rts_hdr, rkey_buf,
+                                     rkey_length);
+    } else {
+        ucp_rndv_receive(worker, rreq, rndv_rts_hdr, rkey_buf);
```

Comment:
protov2 code should not call protov1 code (only vice versa) so move this function to rndv.h and rename to smth without the word proto